comp/core/gui/guiimpl/gui.go

```diff
@@ -124,14 +124,14 @@ func newGui(deps dependencies) provides {
 	sessionExpiration := deps.Config.GetDuration("GUI_session_expiration")
 	g.auth = newAuthenticator(authToken, sessionExpiration)
 
-	router.HandleFunc("/auth", g.getAccessToken).Methods("GET")
-	// Serve the (secured) index page on the default endpoint
-	securedRouter := router.PathPrefix("/").Subrouter()
-	securedRouter.HandleFunc("/", generateIndex).Methods("GET")
-
-	// Mount our (secured) filesystem at the view/{path} route
-	securedRouter.PathPrefix("/view/").Handler(http.StripPrefix("/view/", http.HandlerFunc(serveAssets)))
+	// register the public routes
+	publicRouter.HandleFunc("/", renderIndexPage).Methods("GET")
```

Comment:
`/` used to be on the authenticated handler but now it's public, is that on purpose ?